docs/adr/0015-api-schema.md

```diff
@@ -0,0 +1,64 @@
+# Use schema for API and frontend collaboration
+
+We need a way to agree upon and collaborate on features
+that require changes in the front and back end. Using an
+API schema will allow us to collaborate more quickly
+without breaking things that are working.
+
+## Considered Alternatives
+
+* Use go-swagger with OpenAPI 2.0
```

Comment:
Might want to add a "do nothing" option, where we just read the schema based on the Go payloads and JS requests. Since it's a pretty UI based application (not a public API), it feels like a reasonable alternative.

Reply:
I'm adding it, but given my chat with Isaac and Chris, I think the decided outcome should remain the same.